### PR TITLE
implemented sprint planning and retros

### PR DESCRIPTION
**Pull request by danieljagun**
I added the start of Sprint Planning and Sprint Retros

### CODE REVIEW

**Review by VictorIbikunle — COMMENTED** [review]
spelling error in line 11- Highlight*

File: SprintPlanningAndRetros/SprintPlanning.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+![Planning](sprintplanning.png)
+
+Insights from Resources
+**<u>Estimation Techniques</u>**: Dive deep into how teams can use various techniques to estimate work more accurately, such as time-based estimation, story points, and planning poker. Expanding on the concept of aligning team capabilities and organizing backlog tasks, this discussion introduces techniques to accurately measure the required effort and duration for specific activities.
+
+**<u>Collaboration and Communication</u>**: Highlit the importance of effective communication and collaboration during the sprint planning process. Emphasize how open dialogue, active participation and ensuring that all team members have a voice in the planning process can lead to more engaged teams and better-planned sprints.
```
Comment (CorneliusAkanbi1) [inline SprintPlanningAndRetros/SprintPlanning.md]: highlight spelt wrong

File: SprintPlanningAndRetros/SprintRetrospective.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# What is a Sprint Retrospective?
+The Sprint Retrospective concludes each sprint, offering the team an opportunity to reflect on the sprint that has just ended. The aim of the retrospective is to highlight what worked well and identify challenges faced, encouraging a culture of continuous improvement. By discussing these aspects, the team pinpoints areas for improvement and agrees on actionable steps to enhance future sprints.
+
+![Retro](sprintretro.png)
+
+## Insights from Resources
+**<u>Leveraging Tools and Technologies</u>**: Emphasize the importance of using digital tools and platforms to facilitate more effective and inclusive retrospectives, especially for remote teams. Using these tools can enhance collaboration, allowing team members to contribute equally regardless of their location
+
```
Comment (CorneliusAkanbi1) [inline SprintPlanningAndRetros/SprintRetrospective.md:8]: all around good formatting well done

**Review by CorneliusAkanbi1 — APPROVED** [review]

File: SprintPlanningAndRetros/SprintPlanning.md (hunk 1 above)
Comment (danieljagun) [reply SprintPlanningAndRetros/SprintPlanning.md]: editted error

**Review by VictorIbikunle — APPROVED** [review]

**Pull request merged**